requirements/integration.txt

```diff
@@ -10,8 +10,9 @@ cfgv==3.2.0               # via pre-commit
 click==7.1.2              # via pip-compile-multi, pip-tools
 distlib==0.3.1            # via virtualenv
 filelock==3.0.12          # via tox, virtualenv
-identify==1.5.4           # via pre-commit
-importlib-metadata==1.7.0  # via pluggy, pre-commit, tox, virtualenv
+identify==1.5.6           # via pre-commit
+importlib-metadata==2.0.0  # via pluggy, pre-commit, tox, virtualenv
```

Comment:
It seems `importlib-metadata` is pinned in `base.in`, causing a conflict. Not sure why it's there, but it was probably causing some conflicts earlier. Perhaps try to remove it first, then add it to `integration.in` if it won't work otherwise.

https://github.com/apache/incubator-superset/blob/62629eceab754216a5813ecaf29f461a07afbbbc/requirements/base.in#L20

Reply:
@etr2460 might know something about `importlib-metadata`. It may be pinned whilst we wait for a new version of `tox`.